helm/templates/deployment.yaml

```diff
@@ -49,3 +57,50 @@ spec:
             - "--changeLogFile=changelog/db.changelog-main.yml"
             - "--liquibaseSchemaName=timecard"
             - "update"
+        {{- if eq .Values.environment "dev" }}
```

Comment:
Why do we have separate init containers for each step?

Reply:
May have answered it myself, is it because of the tooling required and we don't have one image with all the tools?

Reply:
update to use a single custom docker image as per our conversation.